src/bisect.jl

```diff
@@ -40,3 +40,38 @@ function bisect(X::IntervalBox, i::Integer, α=where_bisect)
 
     return (X1, X2)
 end
+
+"""
+    mince(x::Interval, n)
+
+Splits `x` in `n` intervals of the same diameter, which are returned
+as a vector.
+"""
+function mince(x::Interval, n)
+    nodes = range(x.lo, x.hi, length = n+1)
+    return [Interval(nodes[i], nodes[i+1]) for i in 1:length(nodes)-1]
+end
+
+"""
+    mince(x::IntervalBox, n)
+
+Splits `x` in `n` intervals in each dimension of the same diameter. These
+intervals are combined in all possible `IntervalBox`-es, which are returned
+as a vector.
+"""
+@generated function mince(x::IntervalBox{N,T}, n) where {N,T}
```

Comment:
Comment: I just noticed that there is also `mince` function defined in src/multidim/intervalbox.jl, essentially with the same arguments. I'm not sure which one should be used nor which is faster.